files/dashboards/openshift-logging-dashboard.json

```diff
@@ -35,6 +35,12 @@
         "#d44a3a"
       ],
       "datasource": "$datasource",
+      "fieldConfig": {
+        "defaults": {
+          "custom": {}
+        },
+        "overrides": []
+      },
```

Comment:
You can remove those empty lines if no need to change them ... not sure why they are added

Reply:
These are coming from Granafa but now I removed them.